app/views/users/edit.html.erb

```diff
@@ -23,6 +23,11 @@
         row.with_key { t('users.index.table_headings.organisation') }
         row.with_value { @user.organisation_slug }
       end
+
+      summary_list.with_row do |row|
+        row.with_key { t('users.index.table_headings.role') }
+        row.with_value { t("users.roles.#{@user.role}.name") }
+      end
```

Comment:
Hi @lfdebrux @deberny, 

Could I ask why are we displaying org & roles as readonly state at the top of the page and then repeating the information again but as a form field? I thought we would have just had the form fields to show the user what is it currently as with any edit form page.

Reply:
![Screenshot 2023-05-04 at 09 31 27](https://user-images.githubusercontent.com/2632224/236151944-376eefa6-90b2-4a5c-af9c-16670a5188bf.png)

We are showing the User details in a summary at the top, playing back what is the current state.

And under organisation we allow the superadmin to change or assign the organisation.
Under role we allow the superadmin to change the user role.

This is not a typical edit page, it's more of a mix of 

- here are the details
- here you can change them

Reply:
Sorry for wanting to dig in a bit deeper to the needs. I'm sure this was already covered in other meetings so happy with what the team have agreed on.

Reply:
It's important to display that **'Organisation is not set'** in the summary and users list column, so that the admin can tell the difference and go in to set it.

Playing back the role as a part of summary list in key-value pattern will make it easier to scan with assistive tech.  I wouldn't want the user to have to rely on navigating to editing fields with 'Radio option checked' to learn what the current value is.

Otherwise we's need to change the page to only playback the read only details, add 'change' action link to the summary list, and build two more pages to 'Edit role' and 'Edit organisation'